docs/sql/statements/create-table.rst

```diff
@@ -385,8 +385,8 @@ automatic background refresh.
 The interval of this background refresh is specified in milliseconds using this
 ``refresh_interval`` setting.
 
-The default is ``null``, which causes tables to be refreshed once every second
-(``1000ms``), but only if the table is not idle. A table can become idle if no
+The default is ``1s``, which causes tables to be refreshed once every second
+but only if the table is not idle. A table can become idle if no
```

Comment:
`null` has different semantics than setting it to `1s`. See the remainder this section.

`RESET` should be used to reset the option and `SET` could probably fail with a pointer towards `RESET`

Reply:
> null has different semantics than setting it to 1s. See the remainder this section.

I have seen the remainder and my understating is there is no fixed default, it's kinda floating between default1s <--> no-refresh ( depending on idle state)

If so, I think it's still fine to let users now actual default since clarification below clearly communicates non-fixed behavior? I mean we don't really operate with NULL, if the only purpose to document floating default maybe we can be more verbose in docs?